docs/conf.py

```diff
@@ -43,6 +43,14 @@
     warninghandler = list(filter(lambda x: isinstance(x, WarningStreamHandler), handlers))[0]
     warninghandler.stream = SafeEncodingWriter(stream=sys.stdout)
 
+# -- I hear you like changelogs in your conf files -----------------------------
```

Comment:
I would just jam this all in one block at the end, no need to put it in the top and the bottom.